src/AuthTrait.php

```diff
@@ -88,6 +88,10 @@ private function getAuthHeaderAttr($attr = [])
             $data[] = 'Redirect-URL: ' . $attr['Redirect-URL'];
         }
 
+        if (isset($attr['Webhook-URL'])) {
```

Comment:
Check for v03 should be added, because this header became available only since 0.3 (see PaymentTrait or AccountTrait)

Reply:
Although, personally, I do not see any problem to pass additional header, platform just won't use it, after seeing code conventions in the library, I will change it.